src/averageAges.js

```diff
@@ -15,11 +15,20 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter((peoples) => peoples.sex === 'm');
+
+  const AverageAge = men.reduce((prev, item) =>
```

Comment:
```suggestion
  const averageAge = men.reduce((prev, item) =>
```
it's not a class, just a variable